resources/views/people/index.blade.php

```diff
@@ -127,33 +127,31 @@
 
                 @foreach($contacts as $contact)
 
-                <li class="people-list-item bg-white">
-                  <a href="{{ route('people.show', $contact) }}">
-                    @if ($contact->has_avatar)
-                      <img src="{{ $contact->getAvatarURL(110) }}" width="43">
+                <li class="people-list-item bg-white pointer" @click="window.location.href='/people/{{ $contact->hashID() }}'">
```

Comment:
Did you consider using a jquery binding `.click()` instead?

Reply:
Yes. I want to get rid of jquery entirely, hence Vue here.